.github/workflows/test.yml

```diff
@@ -64,6 +130,65 @@ jobs:
           distribution: 'temurin'
           cache: 'maven'
 
+      # look for dependencies in maven
+      - name: maven-settings-xml-action
+        uses: whelk-io/maven-settings-xml-action@v21
+        with:
+          repositories: |
+            [
+              {
+                "id": "liquibase",
+                "url": "https://maven.pkg.github.com/liquibase/liquibase",
+                "releases": {
+                  "enabled": "false"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              },
+              {
+                "id": "liquibase-pro",
+                "url": "https://maven.pkg.github.com/liquibase/liquibase-pro",
+                "releases": {
+                  "enabled": "false"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              },
+                {
+                  "id": "liquibase-super-pom",
+                  "url": "https://maven.pkg.github.com/liquibase/liquibase-super-pom",
```

Comment:
The project has been renamed. https://github.com/liquibase/liquibase-parent-pom

Reply:
Yes. Thank you. This will be taken care of in Jandro's PR. I have removed the super-pom from this PR